packages/hoprd/src/index.ts

```diff
@@ -106,6 +106,11 @@ const argv = yargs
     describe: 'For testing local testnets. Prefer local peers to remote.',
     default: false
   })
+  .option('dev', {
```

Comment:
please use a more descriptive argument, for example `--testUseWeakCrypto`

Reply:
changed